### PR TITLE
Automated bdd  docker image

### PR DESCRIPTION
**Pull request by epapbak**
# Description

Add docker image and build_deploy script

## Type of change

N/A

## Testing steps

N/A

## Checklist
* [ ] Pylint passes for Python sources
* [ ] sources has been pre-processed by Black
* [ ] updated documentation wherever necessary

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
```
Comment (ikerreyes) [inline Dockerfile:1]: I think the most recent is ubi9. Maybe we can consider start using it. Or do we have some constrain for it to be 8?

**Review by dpensi — APPROVED** [review]

**Review by tisnik — APPROVED** [review]
seems correct

**Review by ikerreyes — COMMENTED** [review]
Isn't `build_deploy.sh` tight to services onboarded into Clowder?

File: Dockerfile (hunk 1 above)
Comment (epapbak) [reply Dockerfile:1]: I can go with ubi9

**Comment by epapbak** [comment]
> Isn't `build_deploy.sh` tight to services onboarded into Clowder?

I don't think so. We had our services in app-interface for a long time without being onboarded nor using clowder. But I do think it is necessary if I want [this](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/insights/ccx-data-pipeline/build.yml) to build the image and push it to our quay.

**Comment by ikerreyes** [comment]
Sorry, I have not used the right wording. I believe that `build_deploy.sh` is a script used by services in app-interface. I think this repo is not one of them.

**Pull request merged**